cypress/integration/feedback.spec.js

```diff
@@ -0,0 +1,80 @@
+/* global describe, it, beforeEach, cy, expect */
+/* eslint-disable-next-line */
+import React from 'react'
+import fakeData from '../../js/data/report.json'
+import sampleRubric from '../../public/sample-rubric'
+
+describe('Provide Feedback', function () {
+  beforeEach(() => {
+    // cypress currently doesn't handle window.fetch so:
+    // diable browser's fetch this way the fetch polyfill is used which uses XHR
```

Comment:
"disable"

Reply:
@kswenson have you seen the new suggestion option in GitHub, this is a perfect example for it. When making a comment it is the icon that looks like two sheets of paper and a +/- sign.

```suggestion
    // disable browser's fetch this way the fetch polyfill is used which uses XHR
```